### ISSUE TITLE
It's not work when i change Lineseries.DataLabels from true to false

### ISSUE THREAD

**Issue by suibianwsh** [beto-rodriguez/Live-Charts#463]
xaml content:
```
<Window x:Class="Test_LiveCharts.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:Test_LiveCharts"
        xmlns:lvc="clr-namespace:LiveCharts.Wpf;assembly=LiveCharts.Wpf"
        mc:Ignorable="d"
        Title="MainWindow" Height="350" Width="525">
    <Grid>
        <Grid.RowDefinitions>
            <RowDefinition Height="280"/>
            <RowDefinition Height="50"/>
        </Grid.RowDefinitions>
        <lvc:CartesianChart LegendLocation="None">
            <lvc:CartesianChart.Series>
                <lvc:LineSeries Values="{Binding ChartValues}"  DataLabels="True" Name="line"/>
            </lvc:CartesianChart.Series>
            <lvc:CartesianChart.AxisY>
                <lvc:Axis ></lvc:Axis>
            </lvc:CartesianChart.AxisY>
            <lvc:CartesianChart.AxisX>
                <lvc:Axis ></lvc:Axis>
            </lvc:CartesianChart.AxisX>
        </lvc:CartesianChart>
        <StackPanel Grid.Row="1" Orientation="Horizontal" Margin="0,10,0,0">
            <Button Width="150" Margin="10, 0 ,0,0 " Content="set datalabels = false" Click="Button_Click"/>
        </StackPanel>
    </Grid>
</Window>
```
C# code content:
```
    /// <summary>
    /// MainWindow.xaml 的交互逻辑
    /// </summary>
    public partial class MainWindow : Window
    {
        public ChartValues<double> ChartValues { get; set; }

        public MainWindow()
        {
            InitializeComponent();

            int count = 10;
            Random ran = new Random();
            ChartValues = new ChartValues<double>();
            while (count > 0)
            {
                ChartValues.Add(ran.NextDouble());
                count--;
            }

            DataContext = this;
        }

        private void Button_Click(object sender, RoutedEventArgs e)
        {
            line.DataLabels = false; 
        }
    }
```

**Comment by suibianwsh** [comment]
i see the source code of wpfView.LineSeries.
```

public override IChartPointView GetPointView(ChartPoint point, string label)
        {
            var mhr = PointGeometrySize < 10 ? 10 : PointGeometrySize;

            var pbv = (HorizontalBezierPointView) point.View;

            if (pbv == null)
            {
                pbv = new HorizontalBezierPointView
                {
                    Segment = new BezierSegment(),
                    Container = Figure,
                    IsNew = true
                };
            }
            else
            {
                pbv.IsNew = false;
                point.SeriesView.Model.Chart.View
                    .EnsureElementBelongsToCurrentDrawMargin(pbv.Shape);
                point.SeriesView.Model.Chart.View
                    .EnsureElementBelongsToCurrentDrawMargin(pbv.HoverShape);
                point.SeriesView.Model.Chart.View
                    .EnsureElementBelongsToCurrentDrawMargin(pbv.DataLabel);
            }

            if (Model.Chart.RequiresHoverShape && pbv.HoverShape == null)
            {
                pbv.HoverShape = new Rectangle
                {
                    Fill = Brushes.Transparent,
                    StrokeThickness = 0,
                    Width = mhr,
                    Height = mhr
                };

                Panel.SetZIndex(pbv.HoverShape, int.MaxValue);

                var wpfChart = (Chart)Model.Chart.View;
                wpfChart.AttachHoverableEventTo(pbv.HoverShape);

                Model.Chart.View.AddToDrawMargin(pbv.HoverShape);
            }

            if (pbv.HoverShape != null) pbv.HoverShape.Visibility = Visibility;

            if (PointGeometry != null && Math.Abs(PointGeometrySize) > 0.1 && pbv.Shape == null)
            {
                if (PointGeometry != null)
                {
                    pbv.Shape = new Path
                    {
                        Stretch = Stretch.Fill,
                        StrokeThickness = StrokeThickness
                    };
                }

                Model.Chart.View.AddToDrawMargin(pbv.Shape);
            }

            if (pbv.Shape != null)
            {
                pbv.Shape.Fill = PointForeround;
                pbv.Shape.Stroke = Stroke;
                pbv.Shape.StrokeThickness = StrokeThickness;
                pbv.Shape.Width = PointGeometrySize;
                pbv.Shape.Height = PointGeometrySize;
                pbv.Shape.Data = PointGeometry;
                pbv.Shape.Visibility = Visibility;
                Panel.SetZIndex(pbv.Shape, Panel.GetZIndex(this) + 1);

                if (point.Stroke != null) pbv.Shape.Stroke = (Brush) point.Stroke;
                if (point.Fill != null) pbv.Shape.Fill = (Brush) point.Fill;
            }

            if (DataLabels && pbv.DataLabel == null)
            {
                pbv.DataLabel = UpdateLabelContent(new DataLabelViewModel
                {
                    FormattedText = label,
                    Instance = point.Instance
                });
                Panel.SetZIndex(pbv.DataLabel, int.MaxValue - 1);

                Model.Chart.View.AddToDrawMargin(pbv.DataLabel);
            }

            return pbv;
        }

```
So if we want to set LineSeries.DataLabels from true to false,  can't see the datalabes on charts. I think should add some code. such as:
```
.....
if (DataLabels && pbv.DataLabel == null)
            {
                pbv.DataLabel = UpdateLabelContent(new DataLabelViewModel
                {
                    FormattedText = label,
                    Instance = point.Instance
                });
                Panel.SetZIndex(pbv.DataLabel, int.MaxValue - 1);

                Model.Chart.View.AddToDrawMargin(pbv.DataLabel);
            }
            //Controlled by LineSeries.DataLabels can see pbv.DataLabel
            if (pbv.DataLabel != null)
            {
                Visibility vi = Visibility.Visible;
                if (DataLabels == false)
                {
                    vi = Visibility.Hidden;
                }
                pbv.DataLabel.Visibility = vi;
            }
```

**Comment by beto-rodriguez** [comment]
This helps, thanks for the possible solution

**Comment by beto-rodriguez** [comment]
Thanks for the report, I extended the solution to all the series, this fill will be included in the next version (0.9.7)

![togglelabels](https://user-images.githubusercontent.com/10853349/27312586-ed17cd5c-552e-11e7-869e-27679494c72d.gif)

**Comment by suibianwsh** [comment]
thanks ,good luck
